db/migrate/20170530180114_add_manufacturer_and_fru_to_guest_devices.rb

```diff
@@ -0,0 +1,8 @@
+class AddManufacturerAndFruToGuestDevices < ActiveRecord::Migration[5.0]
+  def change
+    add_column :guest_devices, :manufacturer, :string
+    add_column :guest_devices, :fru, :string
```

Comment:
What is `fru`?  If it's "field replaceable unit`, this column name might be better spelled out so as to make it clear to all readers.  Also, why is it a string...I would have expected a boolean.